pkgdiff.pl

```diff
@@ -3232,12 +3253,21 @@ ($$$$$)
     <title>
         $Title
     </title>
+    <link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.datatables.net/1.10.16/css/jquery.dataTables.min.css\" />
```

Comment:
Importing of files from web can be accepted only as an option.

Reply:
Hi @lvc - I've actually updated this branch (as can be seen here: https://github.com/corda/pkgdiff/pull/1). Please ignore this one. I will raise a new PR against lvc/pkgdiff if you're interested in incorporating the changes into the main repo.

Reply:
@lvc I've raised https://github.com/lvc/pkgdiff/pull/31 instead